x-pack/plugins/security_solution/server/cloud_posture/routes/score.ts

```diff
@@ -5,28 +5,22 @@
  * 2.0.
  */
 
-/* eslint-disable @typescript-eslint/no-explicit-any */
 /* eslint-disable  */
 
 import { ElasticsearchClient, Logger } from 'src/core/server';
-import {
-  AggregationsFiltersAggregate,
-  SearchRequest,
-  CountRequest,
-  SearchSort,
-} from '@elastic/elasticsearch/lib/api/types';
+import { SearchRequest, CountRequest } from '@elastic/elasticsearch/lib/api/types';
 import type { SecuritySolutionPluginRouter } from '../../types';
-import { any, string } from 'joi';
+import type { CloudPostureStats, PostureScore } from '../types';
 
 const FINDINGS_INDEX = `kubebeat*`;
 
-const getFindingsEsQuery = (benchmark: string = '*', runId: string): CountRequest => {
+const getFindingsEsQuery = (benchmark: string = '*', cycleId: string): CountRequest => {
```

Comment:
```suggestion
const getFindingsEsQuery = (benchmark = '*', cycleId: string): CountRequest => {
```

note that we don't need to define the type if we're assigning a value that already has it
in this case, `*` is already a string so there's no need to add `string` type to `benchnark`